### PR TITLE
Times replica

### PR DESCRIPTION
**Pull request by balerum03**
We created a replica of the article "Space Ripples Reveal Big Bang’s Smoking Gun" from the New York Times official Web site.

### CODE REVIEW

**Review by christianotieno — CHANGES_REQUESTED** [review]
# **CHANGES REQUIRED** :warning:
Hey @balerum03 & @AdesojiCodeMaster  :wave:
-
You have done a splendid job! :tada:  Keep it up.
We, however, do have some few changes to carry out before we proceed (see review comment).
##
## REQUIRED:
- [ ] the footer is not properly centred. See the difference below 👇 

| [Source](https://www.nytimes.com/2014/03/18/science/space/detection-of-waves-in-space-buttresses-landmark-theory-of-big-bang.html) | [Clone](https://raw.githack.com/balerum03/Newyorktimes/times-replica/index.html) |
|:---:|:---:|
|footer|footer-clone|
## OPTIONAL:
- [ ] You are welcome to validate your CSS using this [tool](https://jigsaw.w3.org/css-validator/validator)
- If you have any questions, let me know here or on slack and I will be glad to clarify with you. You can also reach me and connect with me on my social media or email. :eyes:
##
- When you have made the specified changes, please submit another code review. Thank you!
Happy coding :smile:
**[Christian Otieno](https://github.com/ChristianOtieno)**
[Twitter](https://twitter.com/iamchrisotieno) [linkedin](https://www.linkedin.com/in/christianotieno/) [GitLab](https://gitlab.com/christianotieno)
---
*P.S. all requested changes not marked by [OPTIONAL] or are not positive comments are **REQUIRED***

**Comment by balerum03** [comment]
Footer is now centered as requested.
center-footer

**Review by christianotieno — CHANGES_REQUESTED** [review]
## Status: Changes Required ⚠️ 
Hello, team, we are almost there, just a few tweaks and we should be done here.
- For this project to be approved, the linters must all be passing. Kindly fix that.
- As for the footer, it has not been fully implemented as expected. The footer 
should have a fixed position (it sticks to the bottom part of the screen) ([link to page](https://www.nytimes.com/2014/03/18/science/space/detection-of-waves-in-space-buttresses-landmark-theory-of-big-bang.html?_r=0)).
##
Kindly work on these changes and then submit another code review, Cheers!

File: index.html [hunk 1]
```diff
@@ -0,0 +1,931 @@
...
+                        <a href="#">
+                            <h3>There’s Something Special About the Sun: It’s a Bit Boring</h3>
+                        </a>
+                        <span>April 30</span>
+                    </figure>
+                </div>
+
+
+
+                <div class="editors-pick">
```
Comment (rammazzoti2000) [inline index.html]: Must be an [`<aside>` tag](https://www.w3schools.com/tags/tag_aside.asp) for `Editor's picks` section on the rightmost side. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/positioning_and_floating_elements/aside_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59538403/))

**Review by rammazzoti2000 — CHANGES_REQUESTED** [review]
### Hello @balerum03 I am Alex your code reviewer for this project :raised_hands: 
Great job this far with this project, also implementing the changes required by the previous TSE
The are still some issue to fix before approving this project, see the comments bellow :point_down: 
## :stop_sign: Required Changes :construction: :construction_worker: 
- [ ] There is a lot of repeating codes in our CSS file, let us apply the DRY rule. For more info kindly check this [article](https://speckyboy.com/good-bad-css-practices/) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/section/css_bad_practices_requirement.png) from Pathwright)
- [ ] The navbar should have a fixed position (it sticks to the top part of the screen) ([link to page](https://www.nytimes.com/2014/03/18/science/space/detection-of-waves-in-space-buttresses-landmark-theory-of-big-bang.html?_r=0))
- [ ] The caption for the video it should align following the width of the video, you can see that is overflowing
image

- [ ] You should use [`<aside>` tag](https://www.w3schools.com/tags/tag_aside.asp) for `Editor's picks` section on the rightmost side. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/positioning_and_floating_elements/aside_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59538403/))
### <em>Kindly fix the REQUIRED issues and send another code review request.

Cheers and Happy coding! :clap: 

Code review by Alexandru Bangau

(Feel free to contact me on slack by the handle of alexandru if you have any additional question)</em>
### :stop_sign: Status: Required Changes

File: index.html [hunk 2]
```diff
@@ -0,0 +1,931 @@
...
+<meta name="viewport" content="width= 320, initial-scale= 0.5">
+<meta http-equiv="X-UA-Compatible" content="ie=edge">
+<title>NEWYORKTIMES PAGE</title>
+<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.13.0/css/all.min.css">
+<link rel="stylesheet" href="./assets/style/style.css">
+
+<body>
+
+    <header class="menu-bar">
+        <div class="header-container">
```
Comment (rammazzoti2000) [inline index.html]: The `.header-container` should be a <`nav`> tag and not a div, you should try and write proper HTML semantic code, also the navigation bar should have a fixed position (it sticks to the top of the screen)([link to the page](https://www.nytimes.com/2014/03/18/science/space/detection-of-waves-in-space-buttresses-landmark-theory-of-big-bang.html?_r=0))

**Review by christianotieno — APPROVED** [review]
# PROJECT APPROVED :champagne:
Excellent job on this project guys. :fireworks:  Keep it up.
##
***All the required tasks have been fulfilled in this project. I will go ahead and mark this as complete.***
- Congratulations on the splendid job @balerum03  & @AdesojiCodeMaster! :+1:
- Before we merge this, we can as well validate our CSS using this [tool](https://jigsaw.w3.org/css-validator/validator). I wish you all the very best in your next tasks 🥇 
##
Happy coding :smile:
**[Christian Otieno](https://github.com/ChristianOtieno)**
[Twitter](https://twitter.com/iamchrisotieno) [linkedin](https://www.linkedin.com/in/christianotieno/) [GitLab](https://gitlab.com/christianotieno)

**Pull request merged**